precode.go

```diff
@@ -53,9 +68,18 @@ func main() {
 	chOut := make(chan int64, NumOut)
 
 	var wg sync.WaitGroup
-
 	// 4. Собираем числа из каналов outs
-	// ...
+	for i, out := range outs {
+		wg.Add(1)
+		go func(i int, out <-chan int64,) {
```

Comment:
Лучше внутренние переменные называть по-другому, чем внешние
```suggestion
		go func(i int, in <-chan int64,) {
```

Reply:
```suggestion
		go func(i int, out <-chan int64) {
```